packages/ubuntu_bootstrap/test/test_utils.dart

```diff
@@ -49,3 +53,15 @@ extension WidgetTesterX on WidgetTester {
   StorageService,
 ])
 class _Dummy {} // ignore: unused_element
+
+/// Registers a mock [PageConfigService].
+///
+/// The [pages] argument will override the pages that are returned
+/// (empty by default) if provided.
+void setupMockPageConfig({Map<String, PageConfigEntry> pages = const {}}) {
+  final pageConfigService = MockPageConfigService();
+  registerMockService<PageConfigService>(pageConfigService);
+  when(pageConfigService.pages).thenReturn(pages);
```

Comment:
Misses a stub for `excludedPages` (error is thrown in `installer_test/initializes subiquity`, which somehow doesn't cause a test failure :confused:)